website/docs/release_notes.md

```diff
@@ -5,6 +5,106 @@ sidebar_position: 99
 
 # Release notes
 
+## [v0.1.11](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.11)
+
+### Make `.scalafmt.conf` optional when running the `fmt` command
+Scala CLI can now run the `fmt` command without a `.scalafmt.conf` file present. Previously, if such a file was absent, a `Scalafmt requires explicitly specified version.` error was raised while using the `fmt` command.
+
+The Scala CLI `fmt` command now supports passing the `scalafmt` version and dialect directly from the command line, using the `--scalafmt-dialect` and `--scalafmt-version` options respectively:
+```
+scala-cli fmt --scalafmt-dialect scala3 --scalafmt-version 3.5.8
+```
+Either of those can be skipped, which will make Scala CLI infer a default value:
```

Comment:
NIT: I wonder if explaining it in this much detail is necessary in the release notes, looks slightly verbose.
I'd just end with this sentence and cut the following bullet points.
```suggestion
Either of those can be skipped, which will make Scala CLI infer a default value.
```
...but it can also stay the way it is, I suppose.

Reply:
I agree, this much information could be confusing in Release Notes. I cut the bullet points